pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -177,12 +184,13 @@ def _to_float(x: str) -> float:
         return stats
 
     @staticmethod
-    def _parse_gpu_stats(gpu_ids: str, stats: List[List[float]], keys: List[Tuple[str, str]]) -> Dict[str, float]:
+    def _parse_gpu_stats(device_ids: List[int], stats: List[List[float]], keys: List[Tuple[str,
+                                                                                           str]]) -> Dict[str, float]:
```

Comment:
what happened here ^^

Reply:
Oh, it was auto formatted by pre-commit.com hooks.

Reply:
It should look better by adding this comma

```suggestion
    def _parse_gpu_stats(device_ids: List[int], stats: List[List[float]], keys: List[Tuple[str,
                                                                                           str]],) -> Dict[str, float]:
```